### PR TITLE
finished challenge, whiteboard, and tests

### PR DESCRIPTION
**Pull request by GoldenDog190**
- [ x] Top-level README “Table of Contents” is updated
 - [ x] Feature tasks for this challenge are completed
 - [x ] Unit tests written and passing
     - [ x] “Happy Path” - Expected outcome
     - [x ] Expected failure
     - [x ] Edge Case (if applicable/obvious)
 - [x ] README for this challenge is complete
     - [ x] Summary, Description, Approach & Efficiency, Solution
     - [x ] Link to code
     - [ x] Picture of whiteboard

### CODE REVIEW

File: challenges/lib/src/main/java/challenges/stacksandqueues/PseudoQueue.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package challenges.stacksandqueues;
+
+import java.util.NoSuchElementException;
+
+public class PseudoQueue<T> {
```
Comment (Mattpet26) [inline challenges/lib/src/main/java/challenges/stacksandqueues/PseudoQueue.java:5]: You'll need to go back and look at the challenge. You implemented a normal Queue. Instead of using Node<t>, use Stacks.
